my_project/tab_psy_chart/app_psy_chart.py

```diff
@@ -350,10 +354,19 @@ def update_psych_chart(
     # Add traces
     for i, rh in enumerate(rh_list):
         name = "rh" + str(rh)
+
+        dbt_list_convert = list(dbt_list)
+        rh_convert = list(rh_df[name])
+
+        if si_ip == "ip":
+            for j in range(len(dbt_list)):
+                dbt_list_convert[j] = dbt_list_convert[j]*1.8+32
+            for k in range(len(rh_df[name])):
+                rh_convert[k] = rh_convert[k]*0.0624
         fig.add_trace(
```

Comment:
Since the data here are extracted from the 'pythermalcomfort' (another data source), not the 'df-store'; so I added the additional data conversion method here.